tests/wasm/runtime/index.ts

```diff
@@ -106,6 +107,12 @@ function imports(abi: () => Abi, host: Host): WebAssembly.Imports {
       abort: (message: Pointer, fileName: Pointer, line: number, column: number) => {
         host.abort(abi().readString(message), abi().readString(fileName), line, column)
       },
+      trace: (fileNamePtr: Pointer, from_line: number, to_line: number) => {
+        const fileName = abi().readString(fileNamePtr)
```

Comment:
If this should never be non-null, then maybe we should `assert` so it panics? There is the `readStr` helper that already does this.